{"uid":"27054f298763cb14","name":"test_changing_materialization_changes_relation_type","fullName":"tests.functional.adapter.test_changing_relation_type.TestFireboltChangeRelationTypes#test_changing_materialization_changes_relation_type","historyId":"a90f969832ff26b78f06839c6dd7f853","time":{"start":1723727786875,"stop":1723727786875,"duration":0},"status":"broken","statusMessage":"dbt.exceptions.DbtProfileError: Runtime Error\n  Credentials in profile \"test\", target \"default\" invalid: Runtime Error\n    No dbt_project.yml found at expected path /opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/include/firebolt/dbt_project.yml\n    Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml","statusTrace":"profile = {'account_name': 'firebolt', 'api_endpoint': 'api.staging.firebolt.io', 'database': 'integration_testing__1723727463', 'engine_name': 'integration_testing__1723727463', ...}\nprofile_name = 'test', target_name = 'default'\n\n    @staticmethod\n    def _credentials_from_profile(\n        profile: Dict[str, Any], profile_name: str, target_name: str\n    ) -> Credentials:\n        # avoid an import cycle\n        from dbt.adapters.factory import load_plugin\n    \n        # credentials carry their 'type' in their actual type, not their\n        # attributes. We do want this in order to pick our Credentials class.\n        if \"type\" not in profile:\n            raise DbtProfileError(\n                'required field \"type\" not found in profile {} and target {}'.format(\n                    profile_name, target_name\n                )\n            )\n    \n        typename = profile.pop(\"type\")\n        try:\n>           cls = load_plugin(typename)\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:145: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/factory.py:212: in load_plugin\n    return FACTORY.load_plugin(name)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/factory.py:58: in load_plugin\n    mod: Any = import_module(\".\" + name, \"dbt.adapters\")\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/importlib/__init__.py:127: in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/firebolt/__init__.py:12: in <module>\n    Plugin = AdapterPlugin(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/base/plugin.py:37: in __init__\n    self.project_name: str = project_name_from_path(include_path)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/base/plugin.py:12: in project_name_from_path\n    partial = PartialProject.from_project_root(include_path)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/project.py:571: in from_project_root\n    project_dict = load_raw_project(project_root)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nproject_root = '/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/include/firebolt'\n\n    def load_raw_project(project_root: str) -> Dict[str, Any]:\n    \n        project_root = os.path.normpath(project_root)\n        project_yaml_filepath = os.path.join(project_root, DBT_PROJECT_FILE_NAME)\n    \n        # get the project.yml contents\n        if not path_exists(project_yaml_filepath):\n>           raise DbtProjectError(\n                MISSING_DBT_PROJECT_ERROR.format(\n                    path=project_yaml_filepath, DBT_PROJECT_FILE_NAME=DBT_PROJECT_FILE_NAME\n                )\n            )\nE           dbt.exceptions.DbtProjectError: Runtime Error\nE             No dbt_project.yml found at expected path /opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/include/firebolt/dbt_project.yml\nE             Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/project.py:208: DbtProjectError\n\nThe above exception was the direct cause of the following exception:\n\nunique_schema = 'public'\nproject_root = local('/tmp/pytest-of-runner/pytest-0/project9')\nprofiles_root = local('/tmp/pytest-of-runner/pytest-0/profile9')\nprofiles_yml = {'test': {'outputs': {'default': {'account_name': 'firebolt', 'api_endpoint': 'api.staging.firebolt.io', 'database': 'integration_testing__1723727463', 'engine_name': 'integration_testing__1723727463', ...}}, 'target': 'default'}}\ndbt_project_yml = {'flags': {'send_anonymous_usage_stats': False}, 'name': 'test', 'profile': 'test'}\nclean_up_logging = None\n\n    @pytest.fixture(scope=\"class\")\n    def adapter(\n        unique_schema, project_root, profiles_root, profiles_yml, dbt_project_yml, clean_up_logging\n    ):\n        # The profiles.yml and dbt_project.yml should already be written out\n        args = Namespace(\n            profiles_dir=str(profiles_root),\n            project_dir=str(project_root),\n            target=None,\n            profile=None,\n            threads=None,\n        )\n        flags.set_from_args(args, {})\n>       runtime_config = RuntimeConfig.from_args(args)\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/tests/fixtures/project.py:279: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py:271: in from_args\n    project, profile = cls.collect_parts(args)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py:250: in collect_parts\n    profile = cls.get_profile(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py:106: in get_profile\n    return load_profile(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py:70: in load_profile\n    profile = Profile.render(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:399: in render\n    return cls.from_raw_profiles(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:365: in from_raw_profiles\n    return cls.from_raw_profile_info(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:321: in from_raw_profile_info\n    credentials: Credentials = cls._credentials_from_profile(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nprofile = {'account_name': 'firebolt', 'api_endpoint': 'api.staging.firebolt.io', 'database': 'integration_testing__1723727463', 'engine_name': 'integration_testing__1723727463', ...}\nprofile_name = 'test', target_name = 'default'\n\n    @staticmethod\n    def _credentials_from_profile(\n        profile: Dict[str, Any], profile_name: str, target_name: str\n    ) -> Credentials:\n        # avoid an import cycle\n        from dbt.adapters.factory import load_plugin\n    \n        # credentials carry their 'type' in their actual type, not their\n        # attributes. We do want this in order to pick our Credentials class.\n        if \"type\" not in profile:\n            raise DbtProfileError(\n                'required field \"type\" not found in profile {} and target {}'.format(\n                    profile_name, target_name\n                )\n            )\n    \n        typename = profile.pop(\"type\")\n        try:\n            cls = load_plugin(typename)\n            data = cls.translate_aliases(profile)\n            cls.validate(data)\n            credentials = cls.from_dict(data)\n        except (DbtRuntimeError, ValidationError) as e:\n            msg = str(e) if isinstance(e, DbtRuntimeError) else e.message\n>           raise DbtProfileError(\n                'Credentials in profile \"{}\", target \"{}\" invalid: {}'.format(\n                    profile_name, target_name, msg\n                )\n            ) from e\nE           dbt.exceptions.DbtProfileError: Runtime Error\nE             Credentials in profile \"test\", target \"default\" invalid: Runtime Error\nE               No dbt_project.yml found at expected path /opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/include/firebolt/dbt_project.yml\nE               Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:151: DbtProfileError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"tmpdir_factory","time":{"start":1723727785712,"stop":1723727785712,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"shared_data_dir","time":{"start":1723727785712,"stop":1723727785712,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"test_data_dir","time":{"start":1723727786875,"stop":1723727786876,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clean_up_logging","time":{"start":1723727786876,"stop":1723727786876,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"profiles_root","time":{"start":1723727786876,"stop":1723727786876,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"project_root","time":{"start":1723727786876,"stop":1723727786876,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"profiles_yml","time":{"start":1723727786877,"stop":1723727786878,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"dbt_profile_data","time":{"start":1723727786877,"stop":1723727786877,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"dbt_profile_target","time":{"start":1723727786877,"stop":1723727786877,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"unique_schema","time":{"start":1723727786877,"stop":1723727786877,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"profiles_config_update","time":{"start":1723727786877,"stop":1723727786877,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"prefix","time":{"start":1723727786877,"stop":1723727786877,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"dbt_project_yml","time":{"start":1723727786878,"stop":1723727786879,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"project_config_update","time":{"start":1723727786878,"stop":1723727786878,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"packages_yml","time":{"start":1723727786879,"stop":1723727786879,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"selectors_yml","time":{"start":1723727786879,"stop":1723727786879,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"dependencies","time":{"start":1723727786879,"stop":1723727786879,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"adapter","time":{"start":1723727786879,"stop":1723727786882,"duration":3},"status":"broken","statusMessage":"dbt.exceptions.DbtProfileError: Runtime Error\n  Credentials in profile \"test\", target \"default\" invalid: Runtime Error\n    No dbt_project.yml found at expected path /opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/include/firebolt/dbt_project.yml\n    Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml\n    \n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/tests/fixtures/project.py\", line 279, in adapter\n    runtime_config = RuntimeConfig.from_args(args)\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py\", line 271, in from_args\n    project, profile = cls.collect_parts(args)\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py\", line 250, in collect_parts\n    profile = cls.get_profile(\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py\", line 106, in get_profile\n    return load_profile(\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py\", line 70, in load_profile\n    profile = Profile.render(\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py\", line 399, in render\n    return cls.from_raw_profiles(\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py\", line 365, in from_raw_profiles\n    return cls.from_raw_profile_info(\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py\", line 321, in from_raw_profile_info\n    credentials: Credentials = cls._credentials_from_profile(\n  File \"/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py\", line 151, in _credentials_from_profile\n    raise DbtProfileError(\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"selectors","time":{"start":1723727786879,"stop":1723727786879,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"packages","time":{"start":1723727786879,"stop":1723727786879,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"dependencies_yml","time":{"start":1723727786879,"stop":1723727786879,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"dbt.exceptions.DbtProfileError: Runtime Error\n  Credentials in profile \"test\", target \"default\" invalid: Runtime Error\n    No dbt_project.yml found at expected path /opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/include/firebolt/dbt_project.yml\n    Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml","statusTrace":"profile = {'account_name': 'firebolt', 'api_endpoint': 'api.staging.firebolt.io', 'database': 'integration_testing__1723727463', 'engine_name': 'integration_testing__1723727463', ...}\nprofile_name = 'test', target_name = 'default'\n\n    @staticmethod\n    def _credentials_from_profile(\n        profile: Dict[str, Any], profile_name: str, target_name: str\n    ) -> Credentials:\n        # avoid an import cycle\n        from dbt.adapters.factory import load_plugin\n    \n        # credentials carry their 'type' in their actual type, not their\n        # attributes. We do want this in order to pick our Credentials class.\n        if \"type\" not in profile:\n            raise DbtProfileError(\n                'required field \"type\" not found in profile {} and target {}'.format(\n                    profile_name, target_name\n                )\n            )\n    \n        typename = profile.pop(\"type\")\n        try:\n>           cls = load_plugin(typename)\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:145: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/factory.py:212: in load_plugin\n    return FACTORY.load_plugin(name)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/factory.py:58: in load_plugin\n    mod: Any = import_module(\".\" + name, \"dbt.adapters\")\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/importlib/__init__.py:127: in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/firebolt/__init__.py:12: in <module>\n    Plugin = AdapterPlugin(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/base/plugin.py:37: in __init__\n    self.project_name: str = project_name_from_path(include_path)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/adapters/base/plugin.py:12: in project_name_from_path\n    partial = PartialProject.from_project_root(include_path)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/project.py:571: in from_project_root\n    project_dict = load_raw_project(project_root)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nproject_root = '/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/include/firebolt'\n\n    def load_raw_project(project_root: str) -> Dict[str, Any]:\n    \n        project_root = os.path.normpath(project_root)\n        project_yaml_filepath = os.path.join(project_root, DBT_PROJECT_FILE_NAME)\n    \n        # get the project.yml contents\n        if not path_exists(project_yaml_filepath):\n>           raise DbtProjectError(\n                MISSING_DBT_PROJECT_ERROR.format(\n                    path=project_yaml_filepath, DBT_PROJECT_FILE_NAME=DBT_PROJECT_FILE_NAME\n                )\n            )\nE           dbt.exceptions.DbtProjectError: Runtime Error\nE             No dbt_project.yml found at expected path /opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/include/firebolt/dbt_project.yml\nE             Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/project.py:208: DbtProjectError\n\nThe above exception was the direct cause of the following exception:\n\nunique_schema = 'public'\nproject_root = local('/tmp/pytest-of-runner/pytest-0/project9')\nprofiles_root = local('/tmp/pytest-of-runner/pytest-0/profile9')\nprofiles_yml = {'test': {'outputs': {'default': {'account_name': 'firebolt', 'api_endpoint': 'api.staging.firebolt.io', 'database': 'integration_testing__1723727463', 'engine_name': 'integration_testing__1723727463', ...}}, 'target': 'default'}}\ndbt_project_yml = {'flags': {'send_anonymous_usage_stats': False}, 'name': 'test', 'profile': 'test'}\nclean_up_logging = None\n\n    @pytest.fixture(scope=\"class\")\n    def adapter(\n        unique_schema, project_root, profiles_root, profiles_yml, dbt_project_yml, clean_up_logging\n    ):\n        # The profiles.yml and dbt_project.yml should already be written out\n        args = Namespace(\n            profiles_dir=str(profiles_root),\n            project_dir=str(project_root),\n            target=None,\n            profile=None,\n            threads=None,\n        )\n        flags.set_from_args(args, {})\n>       runtime_config = RuntimeConfig.from_args(args)\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/tests/fixtures/project.py:279: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py:271: in from_args\n    project, profile = cls.collect_parts(args)\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py:250: in collect_parts\n    profile = cls.get_profile(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py:106: in get_profile\n    return load_profile(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/runtime.py:70: in load_profile\n    profile = Profile.render(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:399: in render\n    return cls.from_raw_profiles(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:365: in from_raw_profiles\n    return cls.from_raw_profile_info(\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:321: in from_raw_profile_info\n    credentials: Credentials = cls._credentials_from_profile(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nprofile = {'account_name': 'firebolt', 'api_endpoint': 'api.staging.firebolt.io', 'database': 'integration_testing__1723727463', 'engine_name': 'integration_testing__1723727463', ...}\nprofile_name = 'test', target_name = 'default'\n\n    @staticmethod\n    def _credentials_from_profile(\n        profile: Dict[str, Any], profile_name: str, target_name: str\n    ) -> Credentials:\n        # avoid an import cycle\n        from dbt.adapters.factory import load_plugin\n    \n        # credentials carry their 'type' in their actual type, not their\n        # attributes. We do want this in order to pick our Credentials class.\n        if \"type\" not in profile:\n            raise DbtProfileError(\n                'required field \"type\" not found in profile {} and target {}'.format(\n                    profile_name, target_name\n                )\n            )\n    \n        typename = profile.pop(\"type\")\n        try:\n            cls = load_plugin(typename)\n            data = cls.translate_aliases(profile)\n            cls.validate(data)\n            credentials = cls.from_dict(data)\n        except (DbtRuntimeError, ValidationError) as e:\n            msg = str(e) if isinstance(e, DbtRuntimeError) else e.message\n>           raise DbtProfileError(\n                'Credentials in profile \"{}\", target \"{}\" invalid: {}'.format(\n                    profile_name, target_name, msg\n                )\n            ) from e\nE           dbt.exceptions.DbtProfileError: Runtime Error\nE             Credentials in profile \"test\", target \"default\" invalid: Runtime Error\nE               No dbt_project.yml found at expected path /opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/include/firebolt/dbt_project.yml\nE               Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/dbt/config/profile.py:151: DbtProfileError","steps":[],"attachments":[{"uid":"35861247109b4a94","name":"stdout","source":"35861247109b4a94.txt","type":"text/plain","size":64}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"profiles_yml::0","time":{"start":1723727786941,"stop":1723727786941,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests.functional.adapter"},{"name":"suite","value":"test_changing_relation_type"},{"name":"subSuite","value":"TestFireboltChangeRelationTypes"},{"name":"host","value":"fv-az883-85"},{"name":"thread","value":"2418-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.functional.adapter.test_changing_relation_type"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"27054f298763cb14.json","parameterValues":[]}